simpler_eda/categorical_eda.py

```diff
@@ -0,0 +1,80 @@
+import altair as alt
+import numpy as np
+import pandas as pd
+
+def categorical_plot(
+    data,
+    xval = None,
+    yval = "count",
+    plot_type = "histogram",
+    color = None,
+    title = None,
+    font_size = 10,
+    color_scheme = "yellowgreenblue",
```

Comment:
Should consider tableau20 color scheme instead of the sequential one as we are plotting for different categories.

Reply:
Fixed in latest PR